lib/react/buildModules.js

```
@@ -0,0 +1,77 @@
+const path = require('path');
+const fs = require('fs');
+
+const { ICONS_DIR, REACT_MODULES_DIR } = require('../paths');
+const { getIconNames } = require('../utils/icon');
+const { optimizeSvg } = require('../utils/svg');
+const { transform } = require('../utils/transform');
+
+module.exports = function buildModules(options = {}) {
+  const className = options.className || 'icon';
+
+  const iconNames = getIconNames();
+
+  createModulesDirIfNeeded();
+  const moduleBuildPromises = iconNames.map(name =>
+    buildOptimizedSvgModuleForNameAndClass(name, className));
+
+  return Promise.all(moduleBuildPromises)
+    .then((results) => {
+      console.log(`${results.length} React modules successfully built.`);
+    })
+    .catch((err) => {
+      console.log(`Error building React modules: ${err}`);
```

Should you throw this error as well?